### DISCUSSION TITLE
建议在[附加功能]侧栏中加入B站直播的签到按钮，直播一键签到

### DISCUSSION THREAD

**Discussion by RingingVolcano** [the1812/Bilibili-Evolved#1517] [功能建议 / Ideas]
如题（
有时候懒得看直播又不想错签

**Comment by Geeyun-JY3** [comment]
有这功能的项目真的可以说是多如牛毛，大部分的功能还不少：
- https://github.com/andywang425/BLTH
- https://github.com/RayWangQvQ/BiliBiliTool
- https://github.com/srcrs/BilibiliTask
- https://github.com/metowolf/BilibiliHelper
- https://github.com/JunzhouLiu/BILIBILI-HELPER
